main.js

```diff
@@ -23,6 +23,18 @@ $(document).ready(function () {
     }, 0);
   }
 
+  $('select').on('change', function () {
+    if ($(this).val() == 'epilepsy') {
```

Comment:
Really bad practice with too many ifs. You can make this function more universal so that adding new themes do not require js modifications.